examples/asr/transcribe_speech.py

```diff
@@ -92,6 +93,9 @@ class TranscriptionConfig:
     # Recompute model transcription, even if the output folder exists with scores.
     overwrite_transcripts: bool = True
 
+    # Decoding strategy for CTC models
+    ctc_decoding: CTCCharDecodingConfig = CTCCharDecodingConfig()
```

Comment:
Does it also works for BPE models?

Reply:
Yes, the BPE one is a basic subclass of this. I don't expect them to diverge, so for now this is OK.